indy_node/server/request_handlers/read_request_handler.py

```diff
@@ -1,24 +1,43 @@
 from indy_common.state import domain
+from plenum.common.constants import DATA, TXN_TIME, STATE_PROOF
+from plenum.common.plenum_protocol_version import PlenumProtocolVersion
+from plenum.common.types import f
 
 from plenum.server.request_handlers.handler_interfaces.read_request_handler import \
     ReadRequestHandler as PReadRequestHandler
 
 
 class ReadRequestHandler(PReadRequestHandler):
 
-    def lookup(self, path, isCommitted=True, with_proof=False) -> (str, int):
+    def lookup(self, path, is_committed=True, with_proof=False) -> (str, int):
```

Comment:
What about moving `lookup` and `make_result` to Plenum?
As I can see the only thing is that Plenum doesn't have `decode_state_value`. Maybe we can move it to Plenum as well?

Reply:
done